src/common/commonutils/FileUtils.c

```diff
@@ -1034,15 +1254,77 @@ int GetIntegerOptionFromFile(const char* fileName, const char* option, char sepa
     return result;
 }
 
-int CheckLockoutForFailedPasswordAttempts(const char* fileName, void* log)
+int CheckIntegerOptionFromFileEqualWithAny(const char* fileName, const char* option, char separator, int* values, int numberOfValues, char** reason, void* log)
+{
+    int valueFromFile = -999;
+    int i = 0;
+    int result = ENOENT;
+
+    if ((NULL == values) || (0 == numberOfValues))
+    {
+        OsConfigLogError(log, "CheckIntegerOptionFromFileEqualWithAny: invalid arguments (%p, %u)", values, numberOfValues);
+        return EINVAL;
+    }
+
+    if (-999 != (valueFromFile = GetIntegerOptionFromFile(fileName, option, separator, log)))

```

Comment:
What does -999 mean here?

Reply:
Failure/not found. An integer value that is most unlikely to be returned for real.